contributing-docs/02_how_to_communicate.rst

```diff
@@ -60,9 +60,9 @@ Slack details
 -------------
 
 * The `Apache Airflow Slack <https://s.apache.org/airflow-slack>`_ for:
-   * ad-hoc questions related to development and asking for review (#development channel)
-   * asking for help with first contribution PRs (#development-first-pr-support channel)
-   * troubleshooting (#troubleshooting channel)
+   * ad-hoc questions related to development and asking for review (#contributors channel)
```

Comment:
Maybe instead of related to development use something related to contribution to Airflow.
I worry that someone see development and this one would be a trigger to “I develop my pipeline and something doesn’t work well may I ask in contributors channel”

Reply:
I know this placed into the contributors documentation

Reply:
I think it's okay, we are already deep in the contributing docs here. But good call out.

Reply:
Yeah. Context is important.